src/streamlink/plugins/cnews.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, useragents, validate
+from streamlink.stream import HDSStream, HLSStream, HTTPStream
+
+
+class CNEWS(Plugin):
+    _url_re = re.compile(r'(https|http)://www.cnews.fr(/\w)+')
+    _embed_video_url_re = re.compile(r'class="dm-video-embed_video" src="(?P<dm_url>.*)"')
+    _embed_live_url_re = re.compile(r'class="wrapper-live-player main-live-player"><iframe src="(?P<dm_url>.*)"')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return CNEWS._url_re.match(url)
+
+    def _get_streams(self):
+        # Retrieve URL page and search for Dailymotion URL
+        res = http.get(self.url, headers={'User-Agent' : useragents.CHROME})
```

Comment:
could you remove the whitespace before `:`
for flake8 `E203 — whitespace before ':'`